frontend/src/components/MeetingRoom/Toolbar/Toolbar.tsx

```diff
@@ -72,14 +75,20 @@ const Toolbar = ({
       <div className="flex flex-1 justify-center">
         <AudioControlButton />
         <VideoControlButton />
-        <ScreenSharingButton
-          toggleScreenShare={toggleShareScreen}
-          isSharingScreen={isSharingScreen}
-          isViewingScreenShare={isViewingScreenShare}
-        />
-        <LayoutToggleButton isScreenSharePresent={isScreenSharePresent} />
-        <EmojiGrid />
-        <ArchivingToggle />
+        {isSmallViewport ? (
+          <HiddenToolbarItems />
```

Comment:
I'm not a fan of this name, I think it should be called `PopupMenuToggleButton` or something along those lines - feel free to change completely, but that line of thinking, rather than hidden items.